src/components/MessageList/CustomNotification.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react';
+
+export interface CustomNotificationProps {
+  children: React.ReactNode;
```

Comment:
can import `PropsWithChildren` from React instead of manually specifying children in the type